hack/make/.integration-test-helpers

```diff
@@ -19,16 +19,18 @@ bundle_test_integration_cli() {
 #
 go_test_dir() {
 	dir=$1
-	coverpkg=$2
+	precompiled=$2
+	testbinary="$DEST/test.main"
 	testcover=()
 	testcoverprofile=()
-	testbinary="$DEST/test.main"
 	(
 		mkdir -p "$DEST/coverprofiles"
 		echo '+ go test' $TESTFLAGS "${DOCKER_PKG}${dir#.}"
```

Comment:
I think this echo is now a lie, since it could have happened earlier. I've also found that printing this separately from the actual command is misleading, because it doesn't match the real command.

In a PR (that probably wont merge anytime soon) I removed this and changed it to just `set ix; ... set +x` around the `go test` line. What do you think about making that change here?